vtrace/platforms/win32.py

```diff
@@ -2210,3 +2223,116 @@ def getCacheSyms(self):
             ret.append( (addr-self.loadbase, size, name, stype) )
         return ret
 
+    def genSymbols(self, symopts=None, symtypes=None):
+        '''
+        Generator of symbols... does not cache symbols.  This is good for PDBs of unusual size (POUSes).
+        '''
```

Comment:
thanks!  that's the whole purpose of this PR, to get this bit of discussion and explore different methods to identify a good path forward.  thanks.  hopefully will get to it soon.